### DISCUSSION TITLE
Shipping recent changes

### DISCUSSION THREAD

**Discussion by ndrwrbgs** [morelinq/MoreLINQ#885] [General]
I got worried by the nuget page showing last publish in 2019, but it looks like there's been ongoing development in master.
Is the package now being shipped somewhere other than nuget, or could you help me explain the ship strategy to my team?

**Comment by leandromoh** [comment]
> Is the package now being shipped somewhere other than nuget

AFAIK it does not

> could you help me explain the ship strategy to my team?

probably the "new" commits are minor improvements, not bugs or features; but I agree with you, even after many minor improvements a new minor version of the package could be deployed by @atifaziz

**Comment by viceroypenguin** [comment]
@ndrwrbgs I have released a new version w/ all of the latest changes under morelinq.temp v3.3.3. Hope this helps until @atifaziz  can do a new release

**Comment by atifaziz** [comment]
> Is the package now being shipped somewhere other than nuget

NuGet is still the only place besides a separate [MyGet feed](https://www.myget.org/F/morelinq/api/v3/index.json) for nightly builds coming off the CI runs.

> or could you help me explain the ship strategy to my team?

There is no shipping strategy unfortunately. This is open source and volunteered work, so it mainly depends on the free time and availability of the maintainers/contributors.

**Comment by atifaziz** [comment]
Life happens and so the latest release took a back seat for too long, but for what it's worth, [3.4 is now released](https://github.com/morelinq/MoreLINQ/tree/v3.4.0)!
